src/frontend/magnify/src/components/rooms/MyRooms/MyRooms.tsx

```diff
@@ -1,4 +1,4 @@
-import { Box, Card } from 'grommet';
+import { Box, Card, Text } from 'grommet';
 import { Heading } from 'grommet/components';
```

Comment:
You can squash this commit with the first commit. Anything related to a file all be in the same commit. This is the same feature